test/kafka_connect_data_onboarding_tests/test_data_onboarding.py

```diff
@@ -1,9 +1,41 @@
 import logging
 import sys
+import pytest
+from ..commonsplunk import check_events_from_splunk
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 formatter = logging.Formatter('%(message)s')
 handler = logging.StreamHandler(sys.stdout)
 handler.setFormatter(formatter)
-logger.addHandler(handler)
\ No newline at end of file
+logger.addHandler(handler)
+from ..commonkafka import *
+
+
+class TestDataOnboarding:
+
+    @pytest.fixture(scope='class', autouse=True)
+    def setup_class(self, setup):
+        pytest_configure(setup)
+        yield
+        pytest_unconfigure(setup)
+
+    @pytest.mark.parametrize("test_scenario, test_input, expected", [
+        ("raw_endpoint_no_ack", "NOT chars=*", 2),
+        ("raw_endpoint_with_ack", "NOT chars=*", 2),
+        ("event_endpoint_no_ack", "chars=*", 8),
```

Comment:
this is not safe to change. once we deploy more connectors for different test cases, the event size will increase. please use custom key and value either at data enrichment part or overwrite sourcetype so that you can filter for only events ingested specifically through the connector it is testing for.